ChatApp/Extensions/UIImage+Extension.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  UIImage+Extension.swift
+//  ChatApp
+//
+//  Created by Ana Dzebniauri on 26.04.23.
+//
+
+import UIKit
+
+extension UIImage {
+    static let circleImage = UIImage(systemName: "circle.fill")
+    static let sunImage = UIImage(systemName: "sun.min.fill")
+    static let moonImage = UIImage(systemName: "moon.fill")
```

Comment:
I think using low dash symbol instead of dots would be better. like this: "circle_filled" / "moon_filled" ..

Reply:
i got this UIImages from SF Symbols, so its system name, is it possible to change sf symbol names?

Reply:
oh, ok I got it. then I think you can leave it like this